packages/e2e.sandbox.todowidget/src/secondApp.tsx

```diff
@@ -8,14 +8,12 @@ import {
   Heading,
   Skeleton,
 } from "@radix-ui/themes";
-import React, { useCallback, useEffect, useState } from "react";
+import React, { useEffect } from "react";
 import type Config from "./second.config.js";
 
 export const App: React.FC = () => {
-  const { parameterValues, hostEventTarget } = useFoundryViewContext.withTypes<
-    typeof Config
-  >()();
-  const [newTodoItem, setNewTodoItem] = useState("");
+  const { parameterValues, parameterLoadingState, hostEventTarget } =
+    useFoundryWidgetContext.withTypes<typeof Config>()();
```

Comment:
should this be broken to a higher level to reflect idealized usage?

`const useFoundryWidget = useFoundryWidgetContext.withType<typeof Config>` at the top level?

Reply:
yeah, i was just being lazy lol